PORTAL/jobs/_pyperformance.py

```diff
@@ -2596,7 +2608,7 @@ def compare(
             return None
         filenames = [
             self._filename,
-#            *(os.path.join(cwd, o.filename) for o in others),
+            # *(os.path.join(cwd, o.filename) for o in others),
```

Comment:
Presumably `o.filename` is guaranteed to be an absolute path.  So we can drop this line.